Update README: Governance, Adopters and more links (#4836)
jenkinsci/jenkins: README.md

Our latest and greatest source of Jenkins can be found on [GitHub]. Fork us!

# Contributing to Jenkins

Follow the [contributing guidelines](CONTRIBUTING.md) if you want to propose a change in the Jenkins core.

> For more information about participating in the community  contributing to the Jenkins project,

Reviewer: Broken sentence?

Author: ```suggestion
For more information about participating in the community and contributing to the Jenkins project,
```

see [this page](https://www.jenkins.io/participate/).

Documentation for Jenkins core maintainers is in the [maintainers guidelines](docs/MAINTAINERS.adoc).

# News and Website

> All information about Jenkins can be found on our [website].

Reviewer: Other sections got an empty line between title and first sentence.

Author: ```suggestion

All information about Jenkins can be found on our [website].
```

Follow us on [Twitter](https://twitter.com/jenkinsci) or [LinkedIn](https://www.linkedin.com/company/jenkins-project/).

# Governance

> See the [Jenkins Governance Document](https://www.jenkins.io/project/governance/) for information about the project's open governance, our philosophy and values, and the development practices.

Reviewer: ```suggestion
See the [Jenkins Governance Document](https://www.jenkins.io/project/governance/) for information about the project's open governance, our philosophy and values, and development practices.
```

Our latest and greatest source of Jenkins can be found on [GitHub]. Fork us!

# Contributing to Jenkins

Follow the [contributing guidelines](CONTRIBUTING.md) if you want to propose a change in the Jenkins core.
For more information about participating in the community and contributing to the Jenkins project,
see [this page](https://www.jenkins.io/participate/).

Documentation for Jenkins core maintainers is in the [maintainers guidelines](docs/MAINTAINERS.adoc).

# News and Website

All information about Jenkins can be found on our [website].
Follow us on [Twitter](https://twitter.com/jenkinsci) or [LinkedIn](https://www.linkedin.com/company/jenkins-project/).

# Governance

See the [Jenkins Governance Document](https://www.jenkins.io/project/governance/) for information about the project's open governance, our philosophy and values, and development practices.
Jenkins Code of Conduct can be found [here](https://www.jenkins.io/project/conduct/).

# Adopters

Jenkins is used by millions of users and thousands of companies.

> See [this page] for the list of Jenkins adopters and their success stories.

Reviewer 2: ```suggestion
See [adopters](https://www.jenkins.io/project/adopters/) for the list of Jenkins adopters and their success stories.
```
